recompose-composer/build.gradle.kts

```diff
@@ -9,10 +9,10 @@ repositories {
     mavenCentral()
 }
 dependencies {
-    implementation(project(":recompose-ast"))
+    implementation(project(":recompose-ast","default"))
 
     testImplementation(kotlin("test-junit"))
-    testImplementation(project(":recompose-parser"))
+    testImplementation(project(":recompose-parser","default"))
     testImplementation(project(":recompose-test"))
```

Comment:
What does adding "default" do here?

Reply:
I can't make the recompose-idea module run without it. When i use "runIde" inside Intellij, i always get the error message:


>Could not determine the dependencies of task ':recompose-idea:prepareSandbox'.
> Could not resolve all files for configuration ':recompose-idea:runtimeClasspath'.
   > Could not resolve project :recompose-ast.
     Required by:
         project :recompose-idea > project :recompose-composer
      > The consumer was configured to find a runtime of a library compatible with Java 8, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm'. However we cannot choose between the following variants of project :recompose-ast:
          - compile
          - default
          - runtime
          - testCompile
          - testRuntime
        All of them match the consumer attributes:
          - Variant 'compile' capability recompose:recompose-ast:unspecified declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm':
              - Unmatched attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about how its dependencies are found (required its dependencies declared externally)
                  - Doesn't say anything about its target Java version (required compatibility with Java 8)
                  - Doesn't say anything about its elements (required them packaged as a jar)
                  - Doesn't say anything about its usage (required a runtime)`

Reply:
Interesting. What IntelliJ version are you on? I am on 2020.2.2 CE Anyhow, I'll try if it works with "default" here too and I guess then it can just land that way. :)